src/rb/templates/persistence_impl.erb

```diff
@@ -209,7 +209,12 @@ public class <%= model_defn.impl_name %> extends AbstractDatabaseModel<<%= model
   @Override
   protected <%= model_defn.model_name %> instanceFromResultSet(ResultSet rs, Set<Enum> selectedFields) throws SQLException {
     boolean allFields = selectedFields == null || selectedFields.isEmpty();
-    return new <%= model_defn.model_name %>(rs.getLong("id"),
+    <% if database_defn.disable_rails_id %>
+    long id = -1;
```

Comment:
Yes it will but only if you activate the "quick and dirty" option.
I will disable persistence by default in that case